Display Question, Question ID generator, Data object for Question

jsclose opened the pull request.

Lu-Dai committed:
pop-up box

Lu-Dai committed:
pop-up box 2

Lu-Dai committed:
remove .idea

jsclose committed:
Javascript object, UI, Start Menu, displayquestion

ZixinLiu committed:
sample.css

ZixinLiu committed:
Sample commit

Showing how to commit

ZixinLiu committed:
Recommit Sample.css

Lu-Dai committed:
Add Text

jsclose committed:
Appended gitignore

jsclose committed:
Feat:front end

Created question object.
Created display Question function
Created get random question function given an id
created main driver function

jsclose committed:
Feat: front end

Implemented start menu pop up
Created Category picker start up
Worked with Mark to create question, score, category, answers part.

jsclose committed:
Updated quiz app directory

sshusain committed:
Merge branch 'master' into master

jsclose committed:
Merge pull request #2 from ZixinLiu/master

Merging Css files into directory

jsclose committed:
Merge remote-tracking branch 'SOCR/master' into pr/1

jsclose committed:
Merge pull request #1 from Lu-Dai/master

Reconciliation

jsclose committed:
Merge pull request #3 from jsclose/pr/1

Pr/1

sshusain reviewed (changes requested):
Need to move everything into 1 ready block
May want to implement strict mode
JSON formatting is showing parser error
Should implement sanity checks and validation

jsclose committed:
Question Randomizer

Implemented Fisher-Yates (aka Knuth) Shuffle. Cleaned up some functions
and added some comments

jsclose committed:
Feat:front-end: question shuffler

Implemented yates fisher shuffle algorithm to display the questions in
a random order. Added feature to keep track of new correct answer
section.

jsclose committed:
Implemented Scoreboard

jsclose committed:
feat:working quiz with category selector and scoreboard

Made some values truly global. Created  a next question button that
allows you to display category selector after you answer a question.
Fixed some bugs by making sure to not add category/answers to their
respective list twice

jsclose committed:
Attempt at Json

jsclose committed:
Sanity Checks

jsclose commented:
Updated some sanity checks, still some errors with Json. Updated features, question shuffler, scoreboard, category selector nav, next button, score updater.
